crossenv/__init__.py

```diff
@@ -397,10 +406,18 @@ def create_configuration(self, context):
             sysname = host_info[0]
             machine = host_info[-1]
 
+        release = ''
+        if self.macosx_deployment_target:
+            major, minor = self.macosx_deployment_target.split(".")
+            if major == "10":
+                release = "%s.0.0" % (int(minor) + 4)
+            elif major == "11":
+                release = "%s.0.0" % (int(minor) + 20)
```

Comment:
Catch a ValueError here so we can re-raise with a more helpful error message. If you can handle the error somehow, that's fine too.

Also have an else clause (which can error out if needed) just in case they release "12" earlier than expected 🙂